.remarkrc.mjs

```diff
@@ -1,7 +1,23 @@
-import { createRequire } from 'node:module';
-import { unified } from 'unified';
+/**-*- coding: utf-8 -*- esm -*- /.remarkrc.mjs ********************************
+  *
+  * Source file is part of the OpenINF, Infuse.js, webServagility collection.
+  *
+  ******************************************************************************
+  *
+  * @file Remark Lint custom preset & rule initialization script
+  * @date 9 February 2024
+  * @license MIT/Apache-2.0/BlueOak-1.0.0
+  * @authors Derek Lewis <dereknongeneric@open.inf.is> and other OpenINF Authors
+
```

Comment:
this can pretty much be JSDoc magic comments at this point lol (lemme use a real editor tho)